v1.1/install-cockroachdb.html

```diff
@@ -374,6 +374,13 @@ <h2 id="whats-next">What's Next?</h2>
         <td>Autoconf</td>
         <td>Version 2.68 or higher is required.</td>
       </tr>
+      <tr>
+        <td>Terminfo / NCurses development libraries</td>
```

Comment:
This doesn't wrap too nicely, maybe just `Terminfo/NCurses`?

Reply:
That would be inaccurate. Folk usually already have libncurses installed, but it's not sufficient.